app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,69 @@
+class FoodsController < ApplicationController
+  before_action :set_food, only: %i[show edit update destroy]
```

Comment:
This before_action looks great! Let's just remove the edit and update because we are not working with that in this project.

Reply:
Done